### PR TITLE
refactor(dashboard): rename config constants & directory

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] `Error / Warning / Lint / Type`

### Description

- Rename config constants
- 'ENTIRE' -> 'DOMAIN' / 'SINGLE' -> 'PROJECT'

- Refine directory position. Because it is used commonly at dashboards 
- dashboard-create/config.ts -> dashboards/config.ts 
- dashboard-create/type.ts -> dashboards/type.ts

### CODE REVIEW

File: src/services/dashboards/config.ts [hunk 1]
```diff
@@ -3,6 +3,15 @@ export const refreshInterval = ['off', '15s', '30s', '1m', '5m', '10m', '30m', '
...
 
+export const DASHBOARD_SCOPE = {
+    DOMAIN: 'DOMAIN',
+    PROJECT: 'PROJECT',
+};
+export const DASHBOARD_VIEWER = {
+    PUBLIC: 'PUBLIC',
+    PRIVATE: 'PRIVATE',
+};
+
```
Comment (piggggggggy) [inline src/services/dashboards/config.ts:14]: This types exist in `type.ts` of dashboard store. So, I think we need to delete one. Which one is better?

File: src/services/dashboards/type.ts [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import type { Tags } from '@/models';
 
+import type { DASHBOARD_SCOPE, DASHBOARD_VIEWER } from '@/services/dashboards/config';
 import type { DashboardLayoutWidgetInfo } from '@/services/dashboards/widgets/config';
 
+export type DashboardScope = keyof typeof DASHBOARD_SCOPE;
+export type DashboardViewer = keyof typeof DASHBOARD_VIEWER;
+
```
Comment (piggggggggy) [inline src/services/dashboards/type.ts:8]: Here, too.

**Review by piggggggggy — APPROVED** [review]
LGTM!!
I left small comments.

**Comment by minseolee** [comment]
I checked the commits. Thanks a lot !!!

**Pull request merged**